Python/gc.c

```diff
@@ -1421,7 +1420,7 @@ gc_collect_increment(PyThreadState *tstate, struct gc_collection_stats *stats)
         gc_set_old_space(gc, gcstate->visited_space);
         increment_size += expand_region_transitively_reachable(&increment, gc, gcstate);
     }
-    GC_STAT_ADD(1, objects_queued, region_size);
+    GC_STAT_ADD(1, objects_queued, increment_size);
```

Comment:
I don't know if this is correct, but `region_size` doesn't exist.

Reply:
I'm not sure if this make sense anymore. Maybe just drop the "queued" stat?